dev/CommonStyles/TestUI/CornerRadiusPage.xaml

```diff
@@ -0,0 +1,684 @@
+                <CalendarDatePicker CornerRadius="6"/>
+            </Grid>
+
+            <TextBlock Grid.Row="20" Text="ToggleSwitch" Style="{StaticResource SideHeader}"/>
+            <Grid Grid.Row="20" Style="{StaticResource LeftColumn}">
+                <ToggleSwitch/>
+            </Grid>
+            <Grid Grid.Row="20" Style="{StaticResource RightColumn}">
+                <ToggleSwitch CornerRadius="6"/>
+            </Grid>
+
+            <TextBlock Grid.Row="21" Text="Slider" Style="{StaticResource SideHeader}"/>
+            <Grid Grid.Row="21" Style="{StaticResource LeftColumn}" MinWidth="200">
+                <Slider Orientation="Horizontal"/>
+            </Grid>
+            <Grid Grid.Row="21" Style="{StaticResource RightColumn}" MinWidth="200">
+                <Slider Orientation="Horizontal" CornerRadius="6"/>
+            </Grid>
+
+            <TextBlock Grid.Row="22" Text="Slider" Style="{StaticResource SideHeader}"/>
+            <Grid Grid.Row="22" Style="{StaticResource LeftColumn}">
+                <Slider Orientation="Horizontal" TickFrequency="1" Width="200" TickPlacement="Outside"/>
+            </Grid>
+            <Grid Grid.Row="22" Style="{StaticResource RightColumn}">
+                <Slider Orientation="Horizontal" TickFrequency="1" Width="200" TickPlacement="Outside" CornerRadius="6"/>
+            </Grid>
+
+            <TextBlock Grid.Row="23" Text="" Style="{StaticResource SideHeader}"/>
+            <Grid Grid.Row="23" Style="{StaticResource LeftColumn}" MinHeight="100">
+                <Slider Orientation="Vertical"/>
+            </Grid>
+            <Grid Grid.Row="23" Style="{StaticResource RightColumn}" MinHeight="100">
+                <Slider Orientation="Vertical" CornerRadius="6"/>
+            </Grid>
+
+            <TextBlock Grid.Row="24" Text="" Style="{StaticResource SideHeader}"/>
+            <Grid Grid.Row="24" Style="{StaticResource LeftColumn}">
+                <Slider Orientation="Vertical" TickFrequency="1" Height="100" TickPlacement="Outside"/>
+            </Grid>
+            <Grid Grid.Row="24" Style="{StaticResource RightColumn}">
+                <Slider Orientation="Vertical" TickFrequency="1" Height="100" TickPlacement="Outside" CornerRadius="6"/>
+            </Grid>
+
+            <TextBlock Grid.Row="31" Text="ProgressBar" Style="{StaticResource SideHeader}"/>
+            <Grid Grid.Row="31" Style="{StaticResource LeftColumn}" MinWidth="100">
+                <ProgressBar Minimum="0" Maximum="100" Value="50"/>
+            </Grid>
+            <Grid Grid.Row="31" Style="{StaticResource RightColumn}" MinWidth="100">
+                <ProgressBar Minimum="0" Maximum="100" Value="50" CornerRadius="6"/>
+            </Grid>
+
+            <TextBlock Grid.Row="32" Text="TextBlock" Style="{StaticResource SideHeader}"/>
+            <Grid Grid.Row="32" Style="{StaticResource LeftColumn}">
+                <TextBlock Text="Hello World"/>
```

Comment:
Is this just for completion?  I'm curious why you're including controls that don't actually have a corner radius.

Reply:
To make it clear which controls _don't_ support CornerRadius.
I've used the same list of controls for other controls-wide testing like for the density work. As I said in another comment I'd actually like to drive the list of controls from sort of common code... but didn't have enough desire to tackle that just for this :)

Reply:
Makes sense.